services/kubecost/2.5.2/defaults/cm.yaml

```diff
@@ -239,7 +239,7 @@ data:
         kubernetes.io/ingress.class: kommander-traefik
         ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
         traefik.ingress.kubernetes.io/router.tls: "true"
-        traefik.ingress.kubernetes.io/router.middlewares: "${releaseNamespace}-stripprefixes@kubernetescrd,${releaseNamespace}-forwardauth@kubernetescrd"
```

Comment:
@mhrabovcin why did we drop this here ? I tried accessing UI but couldn't do that so adding this back via #3068 

lmk if i am missing sth here and if there is a better approach.